jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -0,0 +1,225 @@
+# Tests data loading in the Mosviz Jdaviz configuration
+
+import pathlib
+from tempfile import gettempdir
+from zipfile import ZipFile
+
+from astropy.nddata import CCDData
+from astropy.utils.data import download_file
+import numpy as np
+import pytest
+from specutils import Spectrum1D
+
+
+def test_load_spectrum1d(mosviz_helper, spectrum1d):
+    label = "Test 1D Spectrum"
+    mosviz_helper.load_1d_spectra(spectrum1d, data_labels=label)
+
+    assert len(mosviz_helper.app.data_collection) == 2
+    assert mosviz_helper.app.data_collection[0].label == label
+
+    table = mosviz_helper.app.get_viewer('table-viewer')
+    table.widget_table.vue_on_row_clicked(0)
+
+    data = mosviz_helper.app.get_data_from_viewer('spectrum-viewer')
+
+    assert isinstance(list(data.values())[0], Spectrum1D)
+    assert list(data.keys())[0] == label
+
+    with pytest.raises(TypeError):
+        mosviz_helper.load_1d_spectra([1, 2, 3])
+
+
+@pytest.mark.filterwarnings('ignore')
```

Comment:
Since we have complete control of `mos_image` fixture, is it possible to create a test case that is warning free? If not, the warning catching should be localized to where it is really needed to avoid accidentally ignoring warnings we should not be ignoring.

Reply:
I agree; I thought it was strange when I saw this warning. Unfortunately I didn't write these fixtures, and I didn't want this coding refactoring to escalate to larger than a fox hole (😉), so I didn't touch actually trying to reform these fixtures, though I agree we should spend some time refining them

Reply:
In that case, at least make the ignore more specific?